pyvo/mivot/writer/annotations.py

```diff
@@ -146,7 +147,7 @@ def _get_templates(self):
         templates_block += "</TEMPLATES>\n"
         return templates_block
 
-    def build_mivot_block(self, *, templates_id=None):
+    def build_mivot_block(self, *, templates_id=None, no_schema_check=False):
```

Comment:
just nitpicking, but I have a feeling we usually have opt-in kwargs rathe than opt outs, e.g. this could be a `schema_check=True` by default, then the code would be easier to read below as there will be no need for double negation on line L181